### PR TITLE
Add active and closed tab on carrier

### PR DESCRIPTION
**Pull request by uzayr**
https://3.basecamp.com/3194017/buckets/421793/todos/123204149

### CODE REVIEW

File: view/carrier_tree.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<data>
```
Comment (riteshshrv) [inline view/carrier_tree.xml]: this tree view is not required

Comment (uzayr) [inline view/carrier_tree.xml]: But there is no active column in tree.

Comment (riteshshrv) [inline view/carrier_tree.xml]: if records are shown under `Closed` tabs then its obvious that its inactive

File: carrier.xml [hunk 2]
```diff
@@ -16,5 +16,25 @@
...
+            <field name="name">Active</field>
+            <field name="sequence" eval="10"/>
+            <field name="domain">[("active", "=", True)]</field>
+            <field name="act_window" ref="carrier.act_carrier_form"/>
+        </record>
+        <record model="ir.action.act_window.domain" id="act_carrier_form_inactive">
+            <field name="name">Closed</field>
+            <field name="sequence" eval="20"/>
+            <field name="domain">[("active", "=", False)]</field>
+            <field name="act_window" ref="carrier.act_carrier_form"/>
```
Comment (prakashpp) [inline carrier.xml]: add context: <field name="context">{'active_test': False}</field>

**Comment by uzayr** [comment]
screen shot 2016-05-11 at 18 10 29
screen shot 2016-05-11 at 18 10 35

File: carrier.xml [hunk 3]
```diff
@@ -16,5 +16,30 @@
...
+            <field name="context">{"active_test": False}</field>
+        </record>
+        <record model="ir.action.act_window.domain" id="act_carrier_form_active">
+            <field name="name">Active</field>
+            <field name="sequence" eval="10"/>
+            <field name="domain">[("active", "=", True)]</field>
+            <field name="act_window" ref="carrier.act_carrier_form"/>
+        </record>
+        <record model="ir.action.act_window.domain" id="act_carrier_form_inactive">
+            <field name="name">Closed</field>
```
Comment (prakashpp) [inline carrier.xml]: Disabled

**Comment by uzayr** [comment]
screen shot 2016-05-11 at 18 16 49

**Pull request merged**